packages/generator-open-wc/generators/building-webpack/templates/_package.json

```diff
@@ -6,8 +6,10 @@
     "build": "webpack --mode production",
     "build:stats": "webpack --mode production --profile --json > bundle-stats.json"
   },
+  "dependencies": {
+    "@open-wc/building-webpack": "^0.1.3"
```

Comment:
why don't keep it as a devDependency?

Reply:
Ops, moved it to workaround:
<img width="867" alt="screen shot 2019-02-05 at 19 53 41" src="https://user-images.githubusercontent.com/17054057/52296899-c7babc00-297f-11e9-889d-5aec64f57cc0.png">

Reply:
either we add a eslint ignore here or we allow to use dev dependencies for all config files to our eslint config?